docarray/array/storage/weaviate/backend.py

```diff
@@ -76,8 +76,19 @@ def _init_storage(
 
         if isinstance(config.client, str):
             self._client = weaviate.Client(config.client)
+            self._client.batch.configure(
+                batch_size=100,
+                dynamic=True,
+                timeout_retries=5,
+            )
         else:
-            self._client = config.client
+            self._client: weaviate.Client = config.client
+            if self._client.batch.batch_size is None:
+                self._client.batch.configure(
+                    batch_size=100,
+                    dynamic=True,
```

Comment:
What's the meaning of `dynamic=True` ?

Reply:
After the batch creation the `batch_size` would be set to the `recommended` number objects. [Here](https://weaviate-python-client.readthedocs.io/en/latest/weaviate.batch.html#weaviate.batch.Batch) is the docs page, it is CASE III.

Reply:
I see, thanks